src/modules/check_test.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright 2019, Circonus Inc.
+ * All Rights Reserved.
```

Comment:
Reconnoiter is BSD-licensed. New headers should carry some version of the license, probably just with a Circonus copyright.

Reply:
Good point - fixed with same copyright header we are using on other reconnoiter source,

Reply:
fixed